src/extensions/more-data-types.js

```diff
@@ -1,23 +1,25 @@
 // TODO: Replace the empty string in the lines below using Javascript with the correct data types
 
 // 1. Set this variable to be null
-const nullVariable = ''
-
+const nullVariable = null
+console.log(nullVariable)
 // 2. Set this variable to be true
-const trueVariable = ''
-
+const trueVariable = true
+console.log(trueVariable)
 // 2. Set this variable to be the opposite of the trueVariable (ie. false);
-const falseVariable = ''
-
+const falseVariable = false
```

Comment:
`const falseVariable = !trueVariable`

Putting `!` before a boolean will give you the opposite boolean